recipes/pypolyagamma-3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]
+then
+    export CC=gcc CXX=g++
+fi
+
+if [ $(uname) == Linux ]
```

Comment:
Probably should be quoting all of this stuff. Though once `gcc` is on both platforms, the `if` statements can be dropped.